worker/rhpv3.go

```diff
@@ -22,6 +22,10 @@ import (
 )
 
 const (
+	// accountLockingDuration is the time for which an account lock remains
+	// reserved on the bus after locking it.
+	accountLockingDuration = 30 * time.Second
```

Comment:
I'd keep it like this for now to be consistent with the other locking in the bus. Hardcoding it in the bus is a bit less flexible in case we ever want to make it configurable.